doc/source/tune/user-guide.rst

```diff
@@ -295,49 +295,157 @@ When running a hyperparameter search, Tune can automatically and periodically sa
  * use pre-emptible machines (by automatically restoring from last checkpoint)
  * Pausing trials when using Trial Schedulers such as HyperBand and PBT.
 
-To use Tune's checkpointing features, you must expose a ``checkpoint_dir`` argument in the function signature, and call ``tune.checkpoint_dir``:
+Let's cover how to configure your checkpoints storage location, checkpointing frequency, and how to resume from a previous run.
+
+A simple (cloud) checkpointing example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Cloud storage-backed Tune checkpointing is the recommended best practice for both performance and reliability reasons. rsync-based checkpointing is also incompatible with Ray on Kubernetes. If you'd rather checkpoint locally or use rsync based checkpointing, see here.
```

Comment:
Link missing ("here")

also, please add a sentence of context here (explain ssh based rsync checkpointing) - it may not be clear for users that this is the default way. 

I'd also rather frame this positively ("it also enables checkpointing if using Ray on Kubernetes, which does not work out of the box" or so).

Reply:
How do I link to an anchor? "here" would ideally link to the `A simple local/rsync checkpointing example` section below